gossip/blockproc/drivermodule/driver_txs.go

```diff
@@ -178,10 +178,11 @@ func (p *DriverTxListener) OnNewReceipt(tx *types.Transaction, r *types.Receipt,
 }
 
 func effectiveGasPrice(tx *types.Transaction, baseFee *big.Int) *big.Int {
-	if baseFee == nil {
+	gasTip, err := tx.EffectiveGasTip(baseFee)
```

Comment:
Reading the documentation of this function, it is not a 100% replacement of the `EffectiveGasTipValue`. There are cases where the tip could be negative, in which case a negative tip is returned + an error. In the old version, the negative tip would be added to the baseFee, producing the effective gas price. In the new version, the `GasPrice` of the transaction would be used, which is different.

To be equivalent to the past, the error should indeed be ignored. Looking into the implementation of the `EffectiveGasTip` function, it also seems like the only possible error is that the tip might be negative.